udl-list.json

```diff
@@ -37,6 +37,2090 @@
+			"display-name": "ASM for Atmel AVR",
+			"version": "Thu, 07 Jan 2010 00:39:07 GMT",
+			"repository": "",
+			"description": "ASM for Atmel AVR",
+			"author": "Brian Tullio <mailto:btullio@haascnc.com>"
+		},
+		{
+			"id-name": "AsmForAvr2_byArtemZorin",
+			"display-name": "ASM for AVR 2",
+			"version": "Wed, 02 Feb 2011 21:10:08 GMT",
+			"repository": "",
+			"description": "ASM for AVR 2",
+			"author": "Artem Zorin <mailto:A.V.Zorin@ngs.ru>"
+		},
+		{
+			"id-name": "AsmForIntel8051_byJackMaynard",
+			"display-name": "ASM for Intel 8051",
+			"version": "Tue, 31 May 2011 07:12:47 GMT",
+			"repository": "",
+			"description": "ASM for Intel 8051",
+			"author": "Jack Maynard <mailto:jackamaynard@yahoo.com>"
+		},
+		{
+			"id-name": "AsmForMIPS-R2000_byJoaoRicardoRosa",
+			"display-name": "ASM for MIPS R2000",
+			"version": "Thu, 26 Jan 2012 08:32:54 GMT",
+			"repository": "",
+			"description": "ASM for MIPS R2000",
+			"author": "Jo\u006fo Ricardo Rosa <mailto:ashyam.lusco@gmail.com>"
+		},
+		{
+			"id-name": "AsmForPdp11_s11_byElisherer",
+			"display-name": "ASM for PDP-11",
+			"version": "Thu, 31 Dec 2009 21:53:35 GMT",
+			"repository": "",
+			"description": "ASM for PDP-11",
+			"author": "elisherer <mailto:elisherer@users.sourceforge.net>"
+		},
+		{
+			"id-name": "AsmForPIC16F628A_byIceborg",
+			"display-name": "ASM for PIC16F628A",
+			"version": "Wed, 02 Feb 2011 01:02:54 GMT",
+			"repository": "",
+			"description": "ASM for PIC16F628A",
+			"author": "iceborg <mailto:pjm_oliveira@yahoo.com>"
+		},
+		{
+			"id-name": "AsteriskConfig_byBrotherGabriel-Marie",
+			"display-name": "Asterisk Config",
+			"version": "Sun, 24 Nov 2013 13:19:33 GMT",
+			"repository": "",
+			"description": "Asterisk Config",
+			"author": "Brother Gabriel-Marie <mailto:brgabriel@sspx.org>"
+		},
+		{
+			"id-name": "AsteriskSys_byBrotherGabriel-Marie",
+			"display-name": "Asterisk Sys Config",
+			"version": "Sun, 24 Nov 2013 13:19:33 GMT",
+			"repository": "",
+			"description": "Asterisk Sys Config",
+			"author": "Brother Gabriel-Marie <mailto:brgabriel@sspx.org>"
+		},
+		{
+			"id-name": "AutoHotKey_byBrotherGabriel-Marie",
+			"display-name": "AutoHotKey",
+			"version": "Sun, 24 Nov 2013 13:14:07 GMT",
+			"repository": "",
+			"description": "AutoHotKey",
+			"author": "Brother Gabriel-Marie <mailto:brgabriel@sspx.org>"
+		},
+		{
+			"id-name": "AutoLISPforAutoCAD_MichaelPuckett",
+			"display-name": "AutoLISP for AutoCAD",
+			"version": "Thu, 01 Dec 2011 08:50:16 GMT",
+			"repository": "",
+			"description": "AutoLISP for AutoCAD",
+			"author": "Michael Puckett <mailto:puckettm@gmail.com>"
+		},
+		{
+			"id-name": "AVISynth_byRustamMusa",
+			"display-name": "AVISynth",
+			"version": "Sun, 03 Feb 2013 10:32:25 GMT",
+			"repository": "",
+			"description": "AVISynth",
+			"author": "\u041c\u0443\u0441\u0430-\u0410\u0445\u0443\u043d\043e\0432 \u0420\u0443\u0441\u0442\u0430\u043c <mailto:rustam-musa@rambler.ru>"
```

Comment:
The JSON validation failed here.
@pryrt  Are you sure it's a valid string?
@chcg If it's a valid Unicode string, is it possible to modify the build script to validate also Unicode string?

Reply:
@donho, thanks for finding those two.  When they were fixed, it moved past the unicode parsing for the validation.  I'll work on fixing the other validation errors next.

Reply:
as of https://github.com/pryrt/userDefinedLanguages/commit/7f2f5ba6470016c436be48fc0a4d459beabaafa7, validates correctly